include/ruby/internal/error.h

```diff
@@ -50,7 +50,10 @@ typedef enum {
     /** Warning is for experimental features. */
     RB_WARN_CATEGORY_EXPERIMENTAL,
 
-    RB_WARN_CATEGORY_ALL_BITS = (
```

Comment:
Since this file is a public header, we may have to keep the name including the new bit.

Reply:
Ah, I didn't notice that. Thanks, I'll restore the old name.

Reply:
Sorry, I meant to keep the old name and add the new name for the bits enabled by `-w`.

Reply:
Oh. I'll make another PR.